newIDE/app/src/EventsFunctionsExtensionsLoader/EnumerateProperties.js

```diff
@@ -13,6 +13,7 @@ type EnumeratedProperty = {|
   isHidden: boolean,
 |};
 
+// TODO Remove this function as it seems to be a dead code.
```

Comment:
Indeed. Unless the comment is really helpful and prevents developers to use this strategy